Assets/MixedRealityToolkit.Services/TeleportSystem/MixedReality.Toolkit.Services.TeleportSystem.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.MixedReality.Toolkit.Unity.Services.TeleportSystem</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/License.txt</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>The Mixed Reality Toolkit Teleport System provides support comfortable locomotion in your mixed reality applications.</description>
```

Comment:
Typo: _...provides support **for** comfortable..._

Reply:
picking up this change.